src/components/Collection/tabs/associations/Units.js

```diff
@@ -133,6 +130,23 @@ class Units extends PureComponent {
     );
   };
 
+  separateByType = (list) => {
```

Comment:
This is not clear what this function does.
Try to simplify it or write some comments to explain what's going on here.